gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```diff
@@ -53,7 +57,16 @@ fun withMirrors(handler: RepositoryHandler) {
 
 fun normalizeUrl(url: String): String {
     val result = url.replace("https://", "http://")
-    return if (result.endsWith("/")) result else result + "/"
+    return if (result.endsWith("/")) result else "$result/"
+}
+
+if (System.getProperty(PLUGIN_PORTAL_OVERRIDE_URL_PROPERTY) == null && !isEc2Agent() && !isMacAgent()) {
```

Comment:
Previously we can set this property to `https://plugins.gradle.org/m2` to quickly disable the mirror settings if the mirror repo is down. But with init script how can do that quickly? Maybe we need another CI parameter to control this behavior?